src/components/Transactions/TransactionRow.tsx

```diff
@@ -53,6 +45,24 @@ export const TransactionRow = ({ tx, compact }: { tx: Tx; compact?: boolean }) =
     asset => asset.symbol === tx?.tradeDetails?.sellAsset
   )
 
+  let value = tx.value
+  let precision = asset?.precision
+  let txSymbol = symbol
+  let rowTitle = `${tx.type}`
+
+  if (tradeTx) {
+    value = tx?.tradeDetails?.buyAmount || '0'
```

Comment:
please change to `??` over `||` - it's just a monkey see monkey do thing for future devs